CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,116 @@
+# CFPB Open Source Code of Conduct
+
+## Introduction
+
+The [Consumer Financial Protection Bureau](https://www.consumerfinance.gov) (CFPB) is committed to 
+building a safe, welcoming, harassment-free culture for everyone. We do not merely want an 
+environment that is free from hostility; we want one that is actively welcoming and inclusive. We 
+want our team, our workplace culture, and our open source community to reflect and celebrate the 
+diversity of the people we serve.
+
+This Code of Conduct summarizes federal anti-harassment law and CFPB policy. 
+Scope
+
+We expect everyone on the CFPB team, and those contributing to our open source community, to exhibit 
+these behaviors and abide by applicable federal laws and CFPB policies. In addition, we expect 
+everyone within CFPB spaces to exhibit these behaviors and refrain from behavior prohibited by 
+anti-harassment laws and federal policies on harassment. These spaces include:
```

Comment:
It would be nice to simplify/clarify this paragraph in a future revision to this COC.

It seems to be making several different distinctions that might be important:
- CFPB team vs. those contributing to our open source community vs. everyone within CFPB spaces
- "applicable federal laws and CFPB policies" vs. "anti-harassment laws and federal policies on harassment"
- everyone in CFPB spaces vs. [people somehow involved but _not_ in CFPB spaces?]
but it's not clear why these distinctions are made.

Reply:
I agree that this is a bit confusing